libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -107,6 +107,53 @@ string YulUtilFunctions::copyToMemoryFunction(bool _fromCalldata)
 	});
 }
 
+string YulUtilFunctions::copyToMemoryLiteralFunction(string const& _literal)
```

Comment:
Why not name this `copyLiteralToMemory`?

Reply:
Fixed now